src/tests/test_various_bugs.py

```diff
@@ -33,6 +34,15 @@ def make_path(freq):
 #    z = csv_path.read_text(encoding='utf-8')
 #    assert "\0" not in z
 
+# HS did: add a test function for the make_url -- check that the formatting works
```

Comment:
Hayk, 'HS: comment' is enough what there is extra information, you do not have to retell commit history by 'did'